src/app/market/overview-listings/overview-listings.component.html

```diff
@@ -43,8 +43,7 @@
   </div><!-- .control-bar -->
 </app-header>
 
-
-
+<mat-progress-bar mode="indeterminate" *ngIf="isLoading && pages.length < 1"></mat-progress-bar>
```

Comment:
I can't obviously tell why you need _two_ progress bars though.
One that displays if
`isLoading && pages.length < 1`
and another one that displays if
`isLoading && pages.length > 0`
?

Reply:
please check branch, when you initially come in list page progress bar appear at top of list.. and when you scroll down to bottom so its display in the bottom of a page. 
so one appears when no data in list (top progress bar)
another has appeared when you scroll down and load data using n-scroll (bottom progress bar)

Reply:
Cool :+1: